src/Client/components/KatexMermaid.tsx

```
@@ -73,42 +78,40 @@
     )
   }
 
-
-  if ( className?.toLowerCase().startsWith('language-katex')) {
-    return <code className={className} ref={refElement} />
+  if (isKaTeX) {
+    const html = katex.renderToString(code, { throwOnError: false })
+    return <code dangerouslySetInnerHTML={{ __html: html }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)